jvm-libs/metrics/micrometer/src/main/kotlin/net/consensys/linea/metrics/micrometer/MicrometerMetricsFacade.kt

```diff
@@ -63,6 +74,28 @@ class MicrometerMetricsFacade(private val registry: MeterRegistry, private val m
     return MicrometerCounterAdapter(builder.register(registry))
   }
 
+  override fun createHistogram(
+    category: LineaMetricsCategory,
+    name: String,
+    description: String,
+    tags: List<Tag>,
+    baseUnit: String
+  ): Histogram {
+    requireValidMicrometerName(category.toString())
+    requireValidMicrometerName(name)
+    requireValidBaseUnit(baseUnit)
```

Comment:
But where `baseUnit` is used?

Reply:
I'd missed it, the builders `baseUnit` is now being se.t

Reply:
Can you please adjust the test as well so we don't miss it?

Reply:
Added checks for the description and base unit